### PR TITLE
translate: Change translate for admin.config.enable_captcha

### PR DESCRIPTION
**Pull request by a1012112796**
add 'register' to make it more accurate

sadly, it has been misunderstood by Chinese translators, It was translated as "enable login CAPTCHA".😂
image

### CODE REVIEW

**Comment by a1012112796** [comment]
image

image

What happend?🤔 @lunny

**Comment by CirnoT** [comment]
You can always propose change on Crowdin if some translation got it wrong.

**Comment by a1012112796** [comment]
> You can always propose change on Crowdin if some translation got it wrong.

I'm sorry, I haven't check Crowdin before, but after check it , I found an amazing thing , that the translation in cowdin is “启用验证码”，but the translation pushed by crowdin bot is  "启用登录验证码" which is different with cowdin , I wander .... 
This pr is am to change english version form "" Enable CAPTCHA" to "Enable register CAPTCHA" to make it more exact. The english version can't change in crowdin. Thanks.

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -166,7 +166,7 @@ openid_signin = Enable OpenID Sign-In
 openid_signin_popup = Enable user sign-in via OpenID.
 openid_signup = Enable OpenID Self-Registration
 openid_signup_popup = Enable OpenID-based user self-registration.
-enable_captcha = Enable CAPTCHA
+enable_captcha = Enable register CAPTCHA
```
Comment (CirnoT) [inline options/locale/locale_en-US.ini]:
```suggestion
enable_captcha = Enable registration CAPTCHA
```

**Review by CirnoT — CHANGES_REQUESTED** [review]

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (a1012112796) [reply options/locale/locale_en-US.ini]: Thanks

**Review by CirnoT — APPROVED** [review]
I think it's fine

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**